core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -248,7 +251,7 @@ public static String getJournalDirectory(String baseDirectory) {
    */
   public FileSystemMaster(BlockMaster blockMaster, Journal journal) {
     this(blockMaster, journal, ExecutorServiceFactories
-        .fixedThreadPoolExecutorServiceFactory(Constants.FILE_SYSTEM_MASTER_NAME, 2));
+        .fixedThreadPoolExecutorServiceFactory(Constants.FILE_SYSTEM_MASTER_NAME, 3));
```

Comment:
will this cause the file system to spend one more thread after startup consistency check?